app/content/blog/building-beyond-beginner-guitar.md

```diff
@@ -0,0 +1,248 @@
+
+This is a great fully crypto option. However it has limitations. First of all, only those familiar with crypto could use it and it doesn't do anything to solve the 'communication' issue. Yes, it gives me an identity to allow access and that identity could be more directly owned by the user, but it still doesn't afford me a way to 'push' communications. This means email would still be used, because the use of MetaMask login comes with the expectation that most Urbit ID NFT owners don't have running Urbit ships (which may or may not be an accurate assumption).
+
+It also would be dependent on either the hawk-499 upgrade, or building my own more bespoke granular access controls. In exchange, it does enable much greater opportunity with regards to automating actions around payments, though, without needing to integrate with a MEGACORP API such as Stripe, which could be liable to change (or charge me) at any time.
+
+## One-click login
+
+This would add actual security by combining guest comets with email for a one-click login.
+
+For a bit of technical background, when someone visits an Urbit served site, `%eyre` sets a cookie and presents the browser session as a comet. But this _guest_ comet has your `@p` tacked onto the end. ex: `~tillus-nodhus-nidsud-pontug--rammeb-sicpur-nordus-mocwyl`
+
+In effect, it's a fingerprint for that browser that lasts seven days. As long as they don't open up another browser that person, as far as your Urbit is concerned will be that comet for seven days.
+
+The process could look something like this:
++ Purchase the course with Stripe.
++ Use the provided email.
++ Upon opening the course request the email.
++ Send a transactional email with an email service.
++ Open email with one-click login link.
++ Clicking the link maps the email to the relevant guest comet.
++ This will allow access from this browser session for 7 days.
++ After 7 days they'll need another one-click login email.
+
+This requires using an email service that can handle transactional emails, which is a bit of a bummer. But not only would this allow security in my access control, it would also open the door for personalization and progress tracking.
+
+## Service-specific micro hosting provider
+
+Another option I can pursue today attacks the "If everybody was on Urbit, we can make `this` thing, awesome!" problem by, well, onboarding more people on to their own instance of Urbit OS. I could have a dozen of ships running and ready to go. Prebooted, hawk installed, guitar course imported, group membership accepted, and roles assigned.
```

Comment:
Another option I can pursue today addresses the ubiquity problem by onboarding more people onto their own instance of Urbit OS. I could have a dozen pre-configured ships ready: Hawk installed, course imported, group membership accepted, and roles assigned.